test/pkg/test/framework.go

```diff
@@ -528,12 +563,13 @@ func (env *Framework) DeleteAllFrameworkTracedTargetGroups(ctx aws.Context) {
 					g.Expect(err.(awserr.Error).Code()).To(Equal(vpclattice.ErrCodeResourceNotFoundException))
 				}
 			}
-		}).WithTimeout(360 * time.Second).Should(Succeed())
+		}).WithTimeout(8 * time.Minute).Should(Succeed())
```

Comment:
please check body of the "eventually", if err == nil function succeed?

Reply:
For vpc lattice api, delete success could have 2 kind of respone:

- 204 Not content (err==nil) 
- 404 Not Found (err!=nil and err with vpclattice.ErrCodeResourceNotFoundException)
I think both scenarios mean delete success 
<img width="886" alt="image" src="https://github.com/aws/aws-application-networking-k8s/assets/32318664/fb942fd9-4915-4a43-ba1b-9c0311b10de1">

<img width="916" alt="image" src="https://github.com/aws/aws-application-networking-k8s/assets/32318664/df634138-7abf-4955-953e-1a06acd16957">